GITHUB_ORG_MANGEMENT_POLICY.md

```diff
@@ -0,0 +1,93 @@
+# OpenJS GitHub Organization Management Policy
+
+The OpenJS Foundation GitHub Organization (https://github.com/openjs-foundation) is
+provided as a development resource by the OpenJS Foundation under the direction
+of the OpenJS Cross Project Council (CPC).
+
+## OpenJS Admin Repository
+
+The [OpenJS admin repository][openjs-foundation/admin] serves as the
+central location for managing OpenJS GitHub Organization administrative
+activities. Only OpenJS GitHub Organization owners, CPC members have write
+permissions to the OpenJS admin repository.
+
```

Comment:
@joesepi I've cloned your local repo and branched from your branch. Will have some updates PR'd shortly.